app/src/androidTest/java/com/github/sdp/ratemyepfl/dependencyinjection/FakeRepositoryModule.kt

```diff
@@ -16,11 +16,8 @@ abstract class FakeRepositoryModule {
 
     @Singleton
     @Binds
-    abstract fun provideCoursesRepository(repo: FakeCoursesRepository): CoursesRepositoryInterface
+    abstract fun provideCoursesRepository(repo: FakeItemsRepository): ItemsRepositoryInterface
```

Comment:
Did you mean `provideItemsRepository` instead?

Reply:
Oh, yes, nice catch !